extension/tpch/dbgen/build.cpp

```diff
@@ -53,10 +53,10 @@ static void gen_phone(DSS_HUGE ind, char *target, seed_t *seed) {
 	RANDOM(exchg, 100, 999, seed);
 	RANDOM(number, 1000, 9999, seed);
 
-	sprintf(target, "%02d", (int)(10 + (ind % NATIONS_MAX)));
-	sprintf(target + 3, "%03d", (int)acode);
-	sprintf(target + 7, "%03d", (int)exchg);
-	sprintf(target + 11, "%04d", (int)number);
+	snprintf(target, PHONE_LEN, "%02d", (int)(10 + (ind % NATIONS_MAX)));
+	snprintf(target + 3, PHONE_LEN, "%03d", (int)acode);
```

Comment:
if we `snprintf` to `ptr + 3,` isn't `PHONE_LEN` too much `n` by 3?

Reply:
Thanks for pointing out!
I've amended a fix to set the correct buffer size with a few tests: https://onlinegdb.com/XL_2XQdz1